src/lib/Libnet/net_server.c

```diff
@@ -756,28 +724,25 @@ add_conn_data(int sock, void * data)
  *	This function identifies the connection based on index provided
  *  and sets cn_data value
  *
- * @param[in]	sd: socket fd
+ * @param[in]	sd: socket descriptor
  *
- * @return	int - 0 if the connection id corresponding to the socket is found
- * @return	int - Null string, if else
+ * @return pointer to the connection related data
+ * @retval - Null, if sd not found
+ * 
  */
 void *
-get_conn_data(int sock)
+get_conn_data(int sd)
 {
-	int conn_idx = connection_find_actual_index(sock);
-	if (conn_idx < 0) {
-		snprintf(logbuf, sizeof(logbuf), "Could not find index for the socket");
+	int idx = connection_find_actual_index(sd);
+	if (idx < 0) {
+		snprintf(logbuf, sizeof(logbuf), "could not find index for the socket %d", sd);
```

Comment:
I see the usefulness of adding the sd value, but why change capitalization of "could"?

Reply:
I saw most of our error messages (atleast in net_server.c) did not start with Capitals and did not end with a period. Hence to make it similar. However, if you prefer, I can change the log text's back.

Reply:
If the tests pass, I'm fine with the change. I'm simply concerned about regressions. These are the kind of "common sense" changes that should not require an EDD, IMHO.

Reply:
Thanks!